examples/data-objects/inventory-app/webpack.config.cjs

```diff
@@ -38,26 +36,20 @@ module.exports = (env) => {
 			output: {
```

Comment:
```json
						test: /\.m?js$/,
						use: [require.resolve("source-map-loader")],
```
is also likely incorrect. I don't know about source-map-loader, but if `m?` is important, then probably `[cm]?` is correct.

Reply:
I got this from @Abe27342 . Not that I have extensionAlias explicitly for mjs and js, maybe this makes sense.